audmetric/core/api.py

```diff
@@ -179,6 +182,107 @@ def confusion_matrix(
     return matrix
 
 
+def detection_error_tradeoff(
+    truth: typing.Union[
+        typing.Union[int, float],
```

Comment:
According to the docstring this should be `bool` instead of `float` for `truth`: \
 `typing.Union[typing.Union[int, bool], typing.Sequence[typing.Union[int, bool]]]`

Reply:
Nice catch thanks.

BTW, you can use the "Insert a suggestion button" from the buttons above the comment field to propose a change.

Reply:
Ah, I was looking for this. In this case it affects multiple lines. So, maybe better manual.

Reply:
```suggestion
        typing.Union[int, bool],
```

Reply:
>  In this case it affects two lines.

You could also do a suggestion for more than one line. You first have to mark the lines you would like to change, and if you click afterwards the suggestion button it works.

But doesn't matter I will go with your two independent suggestions here.